plugins/idea/pluginImpl/src/main/java/com/imaginea/kodebeagle/impl/util/ScalaImportsUtil.java

```diff
@@ -29,9 +30,12 @@
 import com.intellij.psi.util.PsiTreeUtil;
 import java.util.Map;
 import java.util.Set;
-import org.jetbrains.plugins.scala.lang.psi.impl.ScalaFileImpl;
+// import org.jetbrains.plugins.scala.lang.psi.impl.ScalaFileImpl;
+
+public class ScalaImportsUtil {
+  /* TODO for scala support
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)